optimize/backend/src/main/java/io/camunda/optimize/jetty/LicenseFilter.java

```diff
@@ -90,19 +86,11 @@ private static boolean isLicenseCheckNeeded(
     return !isStaticResource(requestPath)
         && !isRootRequest(requestPath)
         && !isCloudEnvironment(applicationContext)
-        && !isExcludedApiPath(pathInfo)
-        && !isStatusRequest(requestPath);
+        && !isExcludedApiPath(pathInfo);
   }
 
   private static boolean isCloudEnvironment(ApplicationContext applicationContext) {
```

Comment:
rather than return true, why not just delete the method?

Reply:
When its true the method that calls it will correctly always return false, I am not sure if there's a scenario where the other checks could falsely result in true. The method that calls it is also unnecessary since it needs to be false in C8. Basically the entire logic in this class is not needed, but idk if the file will be removed or just adjusted in some way as part of the license topic and didnt want to cause conflicts.

Reply:
Thanks for explaining 👍 
Ok to leave like this as a temporary state knowing that it will be deleted soon